go.mod

```diff
@@ -4,26 +4,28 @@ go 1.17
 
 require (
 	github.com/go-logr/logr v1.2.0
+	github.com/jackc/pgx/v4 v4.15.0
```

Comment:
Note: #32 introduced github.com/lib/pq as the postgres library. Can we update this PR to use that library, or update the root main.go and use the github.com/jackc/pgx/v4 dependency so we can remain consistent with our module dependencies?

Reply:
^ non-blocking/follow-up.